postgresql/doc/Postgresql.md

```diff
@@ -0,0 +1,48 @@
+# Postgresql plugin documentation
+
+This plugin allows MLDB to interact with a PostgreSQL database.
+For more information about PostgreSQL: <https://www.postgresql.org/>
+
+## PostgreSQL import procedure
+
+This procedure allows to import data from a PostgreSQL database into 
+a MLDB dataset.
```

Comment:
I never know if we should use _a_ or _an_ before an acronym starting with a vowel.